DSSV1/index.ts

```diff
@@ -58,7 +58,7 @@ async function run() {
                         if (field.slug) {
                             let name: string = taskParameters.getVariableName(field.slug);
                             let value: string = field.itemValue;
-                            let secret: boolean = taskParameters.isSecret(name);
+                            let secret: boolean = field.isPassword ?? false;
```

Comment:
The isSecret method should be removed as it's no longer used

Reply:
Removed unused method